HACKING.md

```diff
@@ -1 +1 @@
-moved to [contribution guidelines](CONTRIBUTING.md)
\ No newline at end of file
+moved to [contribution guidelines](CONTRIBUTING.md)
```

Comment:
What changed here? oO

Reply:
apparently the existing HACKING.md does not have a new line and I added one here